aggregator/apps/api/v1/stitcher.py

```diff
@@ -47,12 +84,14 @@ class Stitcher:
     def __init__(self, wdiv_resp, wcivf_resp, request):
         self.wdiv_resp = wdiv_resp
         self.wcivf_resp = wcivf_resp
+        self.wcivf_by_ballot = self.make_wcivf_by_ballot()
```

Comment:
Given "by" is a slightly loaded term in this context, would it be reasonable to just make this var `wcivf_ballots` ?